lam4doc/entrypoints/api/handlers.py

```diff
@@ -72,48 +78,44 @@ def upload_rdfs(body: dict, lam_properties_document: FileStorage = None, lam_cla
     """
     logger.debug('Entering upload_rdfs')
 
-    dataset_name = body.get("dataset_name")
-    if not dataset_name:
-        logger.error("The name of the dataset is required.")
-        return 'Dataset name is required', 400
+    if not lam_properties_document and not lam_classes_document and not celex_classes_document:
+        logger.error("No file was specified. Returning 400.")
+        raise BadRequest("Please supply at least one file: lam_properties_document, lam_classes_document, "
+                         "celex_classes_document")
 
+    dataset_name = body.get("dataset_name")
     try:
         logger.debug(body)
-        sparql_adapter = FusekiSPARQLAdapter(LAM_FUSEKI_LOCATION + ":" + str(LAM_FUSEKI_PORT) + "/", requests)
-
-        if lam_properties_document is None and lam_classes_document is None and celex_classes_document is None:
-            logger.error("No file was specified. Returning 400.")
-            return "Please supply at least one file: lam_properties_document, lam_classes_document, " \
-                   "celex_classes_document", 400
+        sparql_adapter = FusekiSPARQLAdapter(config.LAM_FUSEKI_SERVICE, requests)
 
         with tempfile.TemporaryDirectory() as temp_folder:
             if lam_properties_document:
                 local_lam_properties_file = Path(temp_folder) / lam_properties_document.filename
                 lam_properties_document.save(local_lam_properties_file)
                 logger.info("lam_properties_document - saved to " + str(local_lam_properties_file))
-                sparql_adapter.delete_graph(dataset_name, "<"+ LAM_DOCUMENT_PROPERTY_GRAPH + ">")
+                sparql_adapter.delete_graph(dataset_name, "<" + config.LAM_DOCUMENT_PROPERTY_GRAPH + ">")
                 sparql_adapter.upload_file_to_graph(dataset_name,
-                                                    LAM_DOCUMENT_PROPERTY_GRAPH,
+                                                    config.LAM_DOCUMENT_PROPERTY_GRAPH,
                                                     str(local_lam_properties_file))
 
             if lam_classes_document:
                 local_lam_classes_file = Path(temp_folder) / lam_classes_document.filename
                 lam_classes_document.save(local_lam_classes_file)
                 logger.info("lam_classes_document - saved to " + local_lam_classes_file)
                 sparql_adapter.delete_graph(dataset_name,
-                                            "<" + LAM_CLASSES_GRAPH + ">")
+                                            "<" + config.LAM_CLASSES_GRAPH + ">")
```

Comment:
smelly code, this should not be done here, (re)move the bracket handling (to where it belongs)